server/src/main/java/org/candlepin/resource/EventResource.java

```diff
@@ -48,57 +43,57 @@
+    /**
+     * Retrieves a list of Events.
+     *
+     * @deprecated Event persistence/retrieval is being phased out. This endpoint currently returns an
+     * empty list of events, and will be completely removed on the next major release.
+     *
+     * @return an empty list
+     */
+    @Deprecated
+    @ApiOperation(
+        notes = "Retrieves a list of Events. DEPRECATED: Event persistence/retrieval is being phased out. " +
+        "This endpoint currently returns an empty list of events, and will be completely removed " +
+        "on the next major release.",
+        value = "listEvents")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    public List<EventDTO> listEvents() {
-        List<Event> events = eventCurator.listAll().list();
-
-        List<EventDTO> eventDTOs = null;
-        if (events != null) {
-            eventAdapter.addMessageText(events);
-
-            eventDTOs = new ArrayList<>();
-            for (Event event : events) {
-                eventDTOs.add(this.translator.translate(event, EventDTO.class));
-            }
-        }
-
-        return eventDTOs;
+    public List<Event> listEvents() {
+        return Collections.emptyList();
     }
 
-    @ApiOperation(notes = "Retrieves a single Event", value = "getEvent")
-    @ApiResponses({ @ApiResponse(code = 404, message = "") })
+    /**
+     * Retrieves a single Event.
+     *
+     * @param uuid an event uuid
+     *
+     * @deprecated Event persistence/retrieval is being phased out. This endpoint currently always
+     * returns a 400 code, and will be removed on the next major release.
+     *
+     * @return Will not return anything. Instead, throws a {@link BadRequestException}
+     */
+    @Deprecated
+    @ApiOperation(
+        notes = "Retrieves a single Event. DEPRECATED: Event persistence/retrieval is being phased out. " +
+        "This endpoint currently always returns a 400 code, and will be removed " +
+        "on the next major release.",
+        value = "getEvent")
+    @ApiResponses({ @ApiResponse(
+        code = 400,
+        message = "This API endpoint is deprecated, and will be removed on the next major release") })
     @GET
     @Path("{uuid}")
     @Produces(MediaType.APPLICATION_JSON)
-    public EventDTO getEvent(@PathParam("uuid") String uuid) {
-        Event toReturn = eventCurator.get(uuid);
-
-        if (toReturn != null) {
-            List<Event> events = new LinkedList<>();
-
-            events.add(toReturn);
-            eventAdapter.addMessageText(events);
-
-            return this.translator.translate(toReturn, EventDTO.class);
-        }
-
-        throw new NotFoundException(i18n.tr("Event with ID \"{0}\" could not be found.", uuid));
+    public Event getEvent(@PathParam("uuid") String uuid) {
+        throw new BadRequestException(i18n.tr("This API endpoint is deprecated, and will be" +
+            " removed on the next major release.", uuid));
```

Comment:
Special attention to this endpoint change for reviewers: Is it an acceptable solution to throw a 400 here, where the user is fetching a single event (all other event/atom endpoints were changed to return empty lists), or is there a better approach (e.g. should we simply return a non-populated `new Event()`)?

Reply:
To recap the discussion from IRC: use the NotFoundException with the new message and we should be covered here.

Reply:
Fixed